numcodecs/__init__.py

```diff
@@ -44,7 +44,10 @@
     from numcodecs.blosc import Blosc
     register_codec(Blosc)
     # initialize blosc
-    ncores = multiprocessing.cpu_count()
+    try:
+        ncores = multiprocessing.cpu_count()
+    except OSError:
+        ncores = 1
```

Comment:
Adding for reference: We need a `# pragma: no cover` for this block below since we don't have a way to test it really. PR ( https://github.com/zarr-developers/numcodecs/pull/96 ) addresses that.

Reply:
Ah, didn't know about that. Thanks for reporting and fixing it @jakirkham!